src/main/java/Duke/Task/Task.java

```diff
@@ -0,0 +1,53 @@
+package Duke.Task;
+
+/**
+ * Represents an upcoming task in to-do list.
+ */
+public class Task {
+    
+    protected String taskDescription;
+    protected boolean isDone;
+
+    /**
+     * Construct a task item.
+     * @param taskDescription title of the task.
+     * @param isDone status of the task.
+     */
+    public Task(String taskDescription, boolean isDone) {
+        this.taskDescription = taskDescription;
+        this.isDone = isDone;
+    }
+
+    public String getTaskDescription() {
```

Comment:
I'm not really sure, but since this is a public method, it may require a javadoc?